doc/api/esm.md

```diff
@@ -658,150 +675,105 @@ export async function resolve(specifier, context, defaultResolve) {
 }
 ```
 
-#### `getFormat(url, context, defaultGetFormat)`
+#### `load(url, context, defaultLoad)`
 
 > Note: The loaders API is being redesigned. This hook may disappear or its
 > signature may change. Do not rely on the API described below.
 
+> Note: In a previous version of this API, this was split across 3 separate, now
+> deprecated, hooks (`getFormat`, `getSource`, and `transformSource`).
+
 * `url` {string}
 * `context` {Object}
-* `defaultGetFormat` {Function}
+  * `format` {string|null|undefined} The format optionally supplied by the
+  `resolve` hook.
+* `defaultLoad` {Function}
 * Returns: {Object}
   * `format` {string}
+  * `source` {string|ArrayBuffer|TypedArray}
 
-The `getFormat` hook provides a way to define a custom method of determining how
-a URL should be interpreted. The `format` returned also affects what the
-acceptable forms of source values are for a module when parsing. This can be one
-of the following:
+The `load` hook provides a way to define a custom method of determining how
+a URL should be interpreted, retrieved, and parsed.
 
-| `format`     | Description                    | Acceptable Types For `source` Returned by `getSource` or `transformSource` |
+The final value of `format` must be one of the following:
+
+| `format`     | Description                    | Acceptable Types For `source` Returned by `resolve` or `load` |
```

Comment:
```suggestion
| `format`     | Description                    | Acceptable types For `source` returned by `resolve` or `load` |
```

Reply:
Looking at the current docs, there is only 1 other table (of the 668 tables in all the docs) with multi-word headings: [Stream → API for stream implementers](https://nodejs.org/api/stream.html#stream_api_for_stream_implementers), which is sentence-case.

I have no preference and only used Title Case here because it was already title case.

Reply:
I know, not all of these notes are related to your PR; but these are trivial changes that I think we can fix as drive-bys.